packages/core/src/conflict-resolution.ts

```diff
@@ -416,7 +417,10 @@ export class ConflictResolution {
     }
 
     const stateLog = HistoryLog.fromState(this.dispatcher, initialState)
-    const log = await fetchLog(this.dispatcher, tip, stateLog)
+    const log = await fetchLog(this.dispatcher, tip, stateLog).then((log) =>
+      verifyAnchorAndApplyTimestamps(this.logger, this.dispatcher, this.anchorValidator, log)
+    )
```

Comment:
```suggestion
    const logWithoutTimestamps = await fetchLog(this.dispatcher, tip, stateLog)
    const log = await verifyAnchorAndApplyTimestamps(this.logger, this.dispatcher, this.anchorValidator, logWithoutTimestamps)
    )
```

Kind of weird to mix `await` and `.then`.